test/libsolidity/semanticTests/salted_create/salted_create_with_value.sol

```diff
@@ -12,8 +12,8 @@ contract B
 contract A {
     function f() public payable returns (uint, uint, uint) {
         B x = new B{salt: "abc", value: 3}(7);
-        B y = new B{value: 3}{salt: "abc"}(8);
-        B z = new B{value: 3, salt: "abc"}(9);
+        B y = new B{value: 3, salt: "abc"}(8);
+        B z = new B{salt: "abc", value: 3}(9);
```

Comment:
Duplicate line.

Reply:
Here to be the smallest edit from the previous version.